src_python/habitat_sim/utils/common.py

```diff
@@ -14,6 +14,10 @@
 import numpy as np
 import quaternion as qt
 
+from habitat_sim._ext.habitat_sim_bindings.core import (  # noqa: F401
```

Comment:
@erikwijmans Actually, just realized that this should have been imported via `bindings/__init__.py` this should have been caught by the import-linter, but I suppose moving the python to src_python may have broken it.

Reply:
Examples and tests shouldn't, but I think it's fine to important from the bindings directly with src_python. I didn't import this in `bindings/__init__.py` as I think this function's canonical home should be here as this is where we have other rigid transform utils.